iress/xplan/api.py

```diff
@@ -23,10 +25,10 @@ def __init__(self, session: Session, api_path: str):
         self.session = session
         self.api_path = api_path
 
-    def call_content(self):
+    def call_content(self) -> bytes:
         resp = self.call()
         if resp.status_code != 200:
```

Comment:
instead of defining our own exception why not use the standard ones from request?
just call 
```python
resp.raise_for_status()
```